app/scripts/modules/google/src/pipeline/stages/bake/bakeStage.html

```diff
@@ -3,6 +3,15 @@
     <input type="text" class="form-control input-sm"
            ng-model="stage.package"/>
   </stage-config-field>
+  <expected-artifact-multi-selector command="stage"
+                                    ids-field="packageArtifactIds"
```

Comment:
I think this field might need to be registered as an artifact reference In the `config` function of gceBakeStage.js so that it can be successfully cleaned up if artifacts are deleted.  Something like:

```javascript
artifactReferenceServiceProvider.registerReference('stage', () => [
  ['packageArtifactIds'],
]);
```

See `app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestStage.ts` for a comparable example.